android/app/build.gradle

```diff
@@ -202,13 +191,13 @@ android {
             // Due to a bug inside AGP, we have to explicitly set a dependency
             // between configureNdkBuild* tasks and the preBuild tasks.
             // This can be removed once this is solved: https://issuetracker.google.com/issues/207403732
```

Comment:
Can this comment be removed now?

Reply:
Can probably be removed when upgrading the android gradle plugin to 7.3+
but has to stay for now